### DISCUSSION TITLE
useAsyncData data type is of type Ref in template tags

### DISCUSSION THREAD

**Discussion by saidinusah** [nuxt/nuxt#26446] [Questions]
hello guys, when i use the data object in my template tags, the data object is of type Ref. Is there a way to fix this? 
I haven't update my tsconfig and my typescript version in 5.4.3

**Comment by AloisSeckar** [comment]
Show the code snippet please. How do you declare the data and how you reference it in template.

**Comment by saidinusah** [comment]
@AloisSeckar 

const { data, pending, error, refresh } = await useAsyncData(
  'active-customer-list',
  () =>
    customerSearch.search(queryTerm.value, {
      offset: (currentPage.value - 1) * 20,
      sort: ['customerProfileCreationDate:desc'],
      filter: 'status = \'active\' OR status = \'new_user\'',
    }),
  {
    transform: ({ hits, estimatedTotalHits }) => {
      const items = (hits as Array<ICustomerResponse>).map((item) => {
        return {
          customerProfileCreationDate: formatDate(
            item.customerProfileCreationDate ?? '',
            'DD MMMM YYYY hh:mm a',
          ),
          branch: item?.assignedBranch?.branchName
            ? item?.assignedBranch?.branchName.replace('Branch', '')
            : '---',
          agent:
            globalStore.GetAgentName(item.assignedUserKey) === '---'
              ? globalStore.GetAgentName(item.assignedUserKey)
              : '---',
          firstName: item?.firstName,
          lastName: item?.lastName,
          phoneNumber: item?.phoneNumber,
          id: item?.customerId,
          kycLevel: item?.kycLevel || 'MKYC',
          foreNames: `${item.firstName} ${item.otherNames || ''}`,
        }
      })
      return { items, estimatedTotalHits }
    },
    watch: [currentPage],
  },
)




<img width="960" alt="image" src="https://github.com/nuxt/nuxt/assets/38425952/66c7ad71-fbe7-40e5-948a-308a9254685f">
